sickbeard/providers/t411.py

```diff
@@ -48,14 +48,16 @@ def __init__(self):
         self.username = None
         self.password = None
         self.ratio = None
+        self.token = None
+        self.tokenLastUpdate = None
 
         self.cache = T411Cache(self)
 
         self.urls = {'base_url': 'http://www.t411.io/',
-                'search': 'http://www.t411.io/torrents/search/?name=%s&cat=210&subcat=%s&search=%s&submit=Recherche',
-                'login_page': 'http://www.t411.io/users/login/',
-                'download': 'http://www.t411.io/torrents/download/?id=%s',
-                }
+                     'search': 'https://api.t411.me/torrents/search/%s?cid=%s&limit=100',
```

Comment:
The API is still available at api.t411.me, but you're right I should use the .io